Client/Frontend/Browser/BrowserViewController/BrowserViewController+TabToolbarDelegate.swift

```diff
@@ -198,6 +198,11 @@ extension BrowserViewController: TabToolbarDelegate, PhotonActionSheetProtocol {
 
 // MARK: - ToolbarActionMenuDelegate
 extension BrowserViewController: ToolBarActionMenuDelegate {
+    
+    func openURLInCurrentTab(_ url: URL?) {
```

Comment:
Probably nice to add an `// Ecosia` comment since its FF code, like you did below

Reply:
Addressed 💪